### ISSUE TITLE
[Object detection API] Bug in code of testing installation

### ISSUE THREAD

**Issue by nikitaverbis** [tensorflow/models#5363]
Found a bug while testing Object detector API.

**1. How to repeat a bug**
Tensorflow version = branch 1.10 (build from source)
tensorflow/models = last version (23.09.2018).
protoc version = 3.6.1

From `~/<tensorflow_dir>/models/research`
Run command: `python object_detection/builders/model_builder_test.py`

**Error log:**
```
Traceback (most recent call last):
  File "object_detection/builders/model_builder_test.py", line 23, in <module>
    from object_detection.builders import model_builder
  File "<tensorflow_dir>/models/research/object_detection/builders/model_builder.py", line 46, in <module>
    from object_detection.models.ssd_mobilenet_v1_fpn_feature_extractor import SSDMobileNetV1FpnFeatureExtractor
  File "<tensorflow_dir>/models/research/object_detection/models/ssd_mobilenet_v1_fpn_feature_extractor.py", line 38, in <module>
    _CONV_DEFS = _create_modified_mobilenet_config()
  File "<tensorflow_dir>/models/research/object_detection/models/ssd_mobilenet_v1_fpn_feature_extractor.py", line 34, in _create_modified_mobilenet_config
    conv_defs = copy.copy(mobilenet_v1.MOBILENETV1_CONV_DEFS)
AttributeError: 'module' object has no attribute 'MOBILENETV1_CONV_DEFS'
```

**Notes:** At line 124 of file [https://github.com/tensorflow/models/blob/master/research/slim/nets/mobilenet_v1.py](url) you will see:
```
# _CONV_DEFS specifies the MobileNet body
_CONV_DEFS = [
```

**Solution:**
In file /ssd_mobilenet_v1_fpn_feature_extractor.py change 

`conv_defs = copy.copy(mobilenet_v1.MOBILENETV1_CONV_DEFS)`

to 

`conv_defs = copy.copy(mobilenet_v1._CONV_DEFS)`

**Comment by Stroemgren** [comment]
There’s a fix in this pull request #5354
https://github.com/tensorflow/models/pull/5354

**Comment by pkulzc** [comment]
Yes, please use #5354. Otherwise this will be broken again when that PR is merged.
